### ISSUE TITLE
Stem, root and foliage biomass für Validierung

### ISSUE THREAD

**Issue by Rasilgon** [KIT-IfGG/EDE4_0#15]
In Forrester 2021 they used:
Forrester DI, Tachauer IHH, Annighoefer P, Barbeito I, Pretzsch H, Ruiz-Peinado R, Stark H, Vacchiano G, Zlatanov T, Chakraborty T, Saha S, Sileshi GW (2017b) Generalized biomass and leaf area allometric equations for European tree species incorporating stand
structure, tree age and climate. For Ecol Manag 396:160–175

**Comment by Rasilgon** [comment]
Die gültige Formell sind aber in TableA5:
Table A.5. All final species-specific general models fitted to Equations 3-21. CF = correction factor, N = sample size, P-values are for the significance of the given independent variable from anova() in R, VIF = variance inflation factors, PRSE = percent relative standard errors (Equation 25), AIC = Akaike’s information criterion, BIC = Bayesian information criterion, RMSE = square root of the mean square error (Equation 26), RMSEreal = RMSE calculated using the pseudo-observations that include the within equation error, MAPE = mean absolute percentage error (Equatioin 27), Bias = bias calculated using Equation 28, R2 and R2c are a “pseudo” R2 and a conditional “pseudo” R2 calculated using the package MuMIn in R (Bartoń, 2016). Biomass is always in kg, d = diameter (cm), Age = age in years, BA = basal area (m2 ha-1), TPH = trees per hectare, mean annual precipitation is in mm and mean annaul temperature is in degrees celcius. The ranges of each independent variable used to fit the equations are provided to indicate the conditions where the given equations are applicable. Note that when only one equation was available and we could not fit any regional equations the equations can be obtained from Table A.2 (column R), and whether or not there was only one equation can be identified using Figures B.3-B.24. 
References: Bartoń, K. (2016). Multi-Model Inference, R Package ‘MuMIn’ version 1.15.6.

**Comment by Rasilgon** [comment]
Ich habe die Tabelle bereits gepflegt und sie liegt nun in dem SyncShare Ordner (biomassEquation_2017).

**Comment by udeui** [comment]
Super! Ich werde mir das bis morgen anschauen und klären, wo ich noch Klärungsbedarf habe. Es gibt noch ein, zwei mehr Stellen, an denen Klärungsbedarf besteht, beim Klima und beim Wald. Von den Variablen die du vorgeschlagen hast würde ich alle berücksichtigen. Bist du morgen dabei?

**Comment by Rasilgon** [comment]
Ja! Wir können morgen es zusammen anschauen :) Abgesehen davon kannst du deine Frage in den passenden Issue oder in einen New Issue schreiben.
